lib/project_types/extension/commands/create.rb

```diff
@@ -55,22 +49,51 @@ def with_create_form(args)
       end
 
       def supports_development_server?(type)
-        Models::DevelopmentServerRequirements.supported?(type.identifier)
+        Models::DevelopmentServerRequirements.supported?(type)
```

Comment:
Can you update this model to also check for the presence of the binary `ext/shopify-extensions/shopify-extensions`. This way having the feature flag continues to use the old flow unless you actually have the binary installed.